src/views/checkout/AddressInformationView.vue

```diff
@@ -47,6 +58,13 @@ watch(
     () => onSelectedDeliveryAddressChanged()
 )
 
+const _VATIN = ref<string | undefined>(order.value.vatNumber)
```

Comment:
is this consistent with how you named other refs?
looks a bit strange to me

Reply:
No, it's not consistent. I considered `vatIdentificationNumber`, `vatNumber`, `VATIN` and `_VATIN`. I ended up with the last option because I wanted the ref to reflect the real world's acronym which is _VATIN_ but I did not want the ref to start with an uppercase letter. At the end of the day, it's only a ref. Consistency is a beautiful thing, so I will rename it to `vatNumber` -- also, to keep it simple.

Reply:
00fb73f28089e2bd0ced3fa889dac139e14d6745